platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -70,6 +73,8 @@ public class GodotInputHandler implements InputManager.InputDeviceListener {
 	 * Used to decide whether mouse capture can be enabled.
 	 */
 	private int lastSeenToolType = MotionEvent.TOOL_TYPE_UNKNOWN;
+	
+	private static String rotaryInputAxis = AXIS_Y;
```

Comment:
You can remove `static`.

Reply:
No I can not. This property is used in `handleMouseEvent` function which is static.